packages/pharos/src/components/tabs/pharos-tab.ts

```diff
@@ -63,7 +60,9 @@ export class PharosTab extends PharosElement {
   }
 
   private _handleClick(): void {
-    this._triggerSelectedEvent();
+    if (!this.selected) {
+      this._triggerSelectedEvent();
+    }
```

Comment:
i wonder if this conditional check should be here or in the parent 🤔

Reply:
I'm good with cutting off any additional logic as soon as possible as we know its not necessary. I think we could change this to happen higher up as a patch if we notice a problem though.